placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -122,6 +123,9 @@ private void init(@NonNull final Context context, final AttributeSet attrs, fina
                 .setApiClient(PlacesHttpClientResolver.PLACES_HTTP_CLIENT)
                 .setGoogleApiKey(finalApiKey)
                 .build();
+        if (languageCode != null) {
+            api.setLanguageCode(languageCode);
```

Comment:
When a new `api` is set using [`setApi()`](https://github.com/seatgeek/android-PlacesAutocompleteTextView/pull/17/files#diff-6d72b9692d27305a87887ee3d4deed0bR378), we should preserve the language value since we're presenting it as a property of the view.